content/200-concepts/100-components/02-prisma-client/260-client-extensions/100-model.mdx

```diff
@@ -106,3 +106,30 @@ You would call `validate` in your application as follows:
 xprisma.user.validate(someData)
 xprisma.post.validate(someData)
 ```
+
+## Call a custom method from another custom method
+
+You can call a custom method from another custom method, if the two methods are declared on the same model. For example, you can call a custom method on the `user` model from another custom method on the `user` model. It does not matter if the two methods are declared in the same extension or in different extensions.
+
+To do so, use `Prisma.getExtensionContext(this).methodName`. Note that you cannot use the form `prisma.user.methodName`.
```

Comment:
Would be worth explaining, that `prisma` in this case refers to the instance that haven't been extended yet and therefore won't contain `firstMethod`

Reply:
Good point, I'll add this